hugo/content/tutorials/langium_and_monaco.md

```diff
@@ -5,43 +5,35 @@ weight: 6
 
 {{< toc format=html >}}
 
-*Updated on Aug. 2nd, 2023 for usage with monaco-editor-wrapper 2.1.1 & above.*
+*Updated on Oct. 4th, 2023 for usage with monaco-editor-wrapper 3.1.0 & above, as well as Langium 2.0.2*
```

Comment:
We should use at least 3.2.x as the langium generator uses it and the service init was changed between 3.1.0 and 3.2.0. This could prevent some confusion.

Reply:
But not merged, yet. Sorry forgot. Maybe bring this in and make a small update afterwards is better.

Reply:
At this point it may already be in, but we'll see, depends if I can update all usages accordingly before the next update.

Reply:
If possible, I would make 3.3.0 the common baseline (minilogo?, this and generator-langium).

Reply:
Given minilogo itself hasn't been updated yet to 3.3.0, this feels like another cycle (albeit smaller). We can do another cycle to update MiniLogo with the new wrapper after, and then the docs as well, which shouldn't be bad. It will also ensure we get the docs updated sooner rather than later (we haven't used the approach outlined here for some time now, so we should get that up to date).